### PR TITLE
Better interfaces

### PR DESCRIPTION
**Pull request by refs**
- micro store interface does not really match our needs so we defined our own
- accounts service definition in protobuf
- micro cli@v2

### CODE REVIEW

File: pkg/command/server.go [hunk 1]
```diff
@@ -3,29 +3,51 @@ package command
...
 	"github.com/oklog/run"
+	"github.com/owncloud/ocis-accounts/pkg/config"
 	"github.com/owncloud/ocis-accounts/pkg/micro/grpc"
 )
 
 // Server is the entry point for the server command.
-func Server() cli.Command {
-	return cli.Command{
+func Server(cfg *config.Config) *cli.Command {
+	baseDir, _ := filepath.Abs(filepath.Dir(os.Args[0]))
```
Comment (tboerger) [inline pkg/command/server.go]: PLEASE NEVER do such a stupid default, it should be NEVER in the same dir as the binary.

Comment (refs) [reply pkg/command/server.go]: Yeah, not happy about this one. What would you set as a default location that works in different OS? 🤔

Comment (tboerger) [reply pkg/command/server.go]: This depends on the user AND the os. Not sure if a default is that great at all.

Comment (refs) [reply pkg/command/server.go]: Actually no default os better, since different implementations might not care about this path.

**Pull request merged**